<?xml version="1.0"?>
<launch>
  <!-- ground truth to tf publisher -->
  <node pkg="active_perception_evaluation" type="floating_kinect_localizer" name="floating_kinect_localizer" respawn="true" output="screen" />
  <!-- regular map for bootstrapping of octomap -->
  <node name="map_server" pkg="map_server" type="map_server" args="$(find active_perception_evaluation)/config/map_sim.pgm 0.05" respawn="true" output="screen" />
</launch>
